core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/kafka/update-properties.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Is server.properties file exists
+  stat:
+    path: "{{ specification.kafka_var.conf_dir }}/server.properties"
+
+- name: Is defined current_kafka_version property
+  shell: grep "^CURRENT_KAFKA_VERSION" {{ specification.kafka_var.conf_dir }}/server.properties
+  register: current_kafka_version_property
+  ignore_errors: yes
+
+- name: Add current_kafka_version property
+  lineinfile:
+    path: "{{ specification.kafka_var.conf_dir }}/server.properties"
+    line: "CURRENT_KAFKA_VERSION={{ specification.kafka_var.version }}"
+  when: current_kafka_version_property.stdout == ""
+
+- name: Is defined inter.broker.protocol.version property
+  shell: grep "^inter.broker.protocol.version" {{ specification.kafka_var.conf_dir }}/server.properties
+  register: inter_broker_protocol_version_property
+  ignore_errors: yes
```

Comment:
Please do not use `ignore_errors` but `failed_when` with `rc` or `stdout` or `stderr` based condition

Reply:
Added:

  failed_when:
    - result.rc == 2